src/background/controllers/rpcController.ts

```diff
@@ -20,6 +20,8 @@ export default class RPCController extends IController {
   * @param abi The ABI of the contract.
   * @param methodName The method to call that is in the ABI.
   * @param args The arguments that are needed when calling the method.
+  *   @argParam gasLimit (unit - gas)
+  *   @argParam gasPrice (unit - satoshi/gas)
```

Comment:
is this standard? never seen this way before.

Reply:
I made it up, open to alternatives, I just wanted to have the units specified